### PR TITLE
Show current focused class in UI [Using Bindings]

### LINKED ISSUE #72: As a user, I can know whether or not I am in currently in focus mode
...so that I know whether or not the current list of students is class-specific

### PR DESCRIPTION
**Pull request by RezwanArefin01**
Resolves #72 using bindings and string property. Alternate to #76. 

Implementation: 
* Add a label to show current focus in `ResultDisplay`.
* Store a `SimpleStringProperty` in `ModelManager` for the focus label.
* Add a getter in `Logic` that is used forward the `SimpleStringProperty` from `ModelManager` to `MainWindow`. 
* Bind the label to the string property in `MainWindow` when it is being initialized.

### CODE REVIEW

File: src/main/java/seedu/taassist/model/ModelManager.java [hunk 1]
```diff
@@ -19,11 +20,15 @@
  * Represents the in-memory model of TA-Assist data.
  */
 public class ModelManager implements Model {
+    private static final String DEFAULT_FOCUS_LABEL = "";
```
Comment (Bubbl3T) [inline src/main/java/seedu/taassist/model/ModelManager.java:23]: I think we can see how "" looks like. If it looks like something is missing we can change it to "-" afterwards.

File: src/main/resources/view/DarkTheme.css [hunk 2]
```diff
@@ -11,8 +11,8 @@
 }
 
 .label-bright {
-    -fx-font-size: 11pt;
-    -fx-font-family: "Segoe UI Semibold";
+    -fx-font-size: 10pt;
+    -fx-font-family: "Segoe UI Semilight";
```
Comment (Bubbl3T) [inline src/main/resources/view/DarkTheme.css:15]: Yay we get nice fonts too! 👍🏻

**Review by Bubbl3T — COMMENTED** [review]
The changes look good! I am also thinking, we might be able to keep only the `focusLabelProperty` and use that in place of `focusedClass`. This is a possible code enhancement to be considered in the future.

**Review by Berted — APPROVED** [review]
LGTM!

File: src/main/java/seedu/taassist/model/ModelManager.java (hunk 1 above)
Comment (RezwanArefin01) [reply src/main/java/seedu/taassist/model/ModelManager.java:23]: @Bubbl3T This is not the content inside "[]". By "Focus Label" I am referring to the entire focus label: "Focus: [ClassName]". So making it "" removes the whole thing. Which is the default when not in focus.

**Pull request merged**